Client/src/components/Pages/Profile/Profile.jsx

```
@@ -0,0 +1,483 @@
+    return (
+      <div className="h-screen w-full flex items-center justify-center bg-gray-900">
+        <Spinner size="xl" color="info" />
+      </div>
+    );
+  }
+
+  return (
+    <div className="min-h-screen w-full flex flex-col items-center py-6 px-4 bg-gray-900 relative">
+      {/* Background blur elements */}
+      <div className="absolute top-0 left-0 w-full h-full overflow-hidden z-0">
+        <div className="absolute top-10 -left-40 w-96 h-96 bg-blue-700 opacity-20 rounded-full filter blur-3xl"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)